geopackage_validator/cli.py

```diff
@@ -148,7 +148,7 @@ def geopackage_validator_command(
 
     if gpkg_path is None and s3_endpoint_no_protocol is None:
         logger.error("Give --gpkg-path or s3 location")
-        return
+        sys.exit(1)
```

Comment:
Does this result in a valid json output?
It would be nice to see the gpkg-validator as a 'webservice on commandline' such that every 'request' (command) always results in a correctly formatted response (json return value). In HTTP the client would send an accept-header, here JSON would be implied.

Reply:
No it doesn't and it didn't. To take the webservice on commandline analogy a bit further: when the web _server_ for the webservice is incorrectly configured it doesn't start and throws an error. That is the behaviour I propose in this pull request.

Reply:
With a correctly configured validator it behaves as you want (it will always respond with json).